projects/angular/src/data/tree-view/_tree-view.clarity.scss

```
@@ -46,6 +45,69 @@
       tree-view-variables.$clr-tree-node-letter-spacing,
       variables.$clr-use-custom-properties
     );
+
+    &:disabled {
+      @include mixins.css-var(
+        color,
+        clr-tree-node-content-disabled-color,
+        tree-view-variables.$clr-tree-node-caret-link-disabled-color,
+        variables.$clr-use-custom-properties
+      );
+    }
```

Add both :disabled and .disabled to support both cases. Although I think that should be put on `clr-tree-node` instead because that's the main item and I would presume if someone is disabling something it would be the tree node, not just the content of it.
```suggestion
    &:disabled,
    &.disabled {
      @include mixins.css-var(
        color,
        clr-tree-node-content-disabled-color,
        tree-view-variables.$clr-tree-node-caret-link-disabled-color,
        variables.$clr-use-custom-properties
      );
    }
```

---

Separate JIRA CDE-1054 for disabled functionality is created.